packages/swapper/src/swappers/thorchain/utils/bitcoin/utils/getThorTxData.ts

```diff
@@ -46,14 +45,16 @@ export const getThorTxInfo: GetBtcThorTxInfo = async ({
       `${deps.midgardUrl}/thorchain/inbound_addresses`
     )
 
-    const btcInboundAddresses = inboundAddresses.find((inbound) => inbound.chain === 'BTC')
+    const sellAssetInboundAddresses = inboundAddresses.find(
+      (inbound) => inbound.chain === sellAsset.symbol.toUpperCase()
+    )
 
-    const vault = btcInboundAddresses?.address
+    const vault = sellAssetInboundAddresses?.address
 
     if (!vault)
-      throw new SwapError(`[getThorTxInfo]: vault not found for BTC`, {
+      throw new SwapError(`[getThorTxInfo]: vault not found for asset`, {
```

Comment:
Add asset, or change this from a template literal to a string?

Reply:
Not sure I follow this comment? 

It attaches `details` to the error with asset information:

```
        details: { inboundAddresses, sellAsset }
 ```

Reply:
We don't interpolate any values, so it should be a string literal rather than a template literal.

Reply:
This is more of a stylistic opinion. In my opinion its just fine as it is.

Also it builds into the exact same thing.